monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -26,9 +29,11 @@ class SSHExploiter(HostExploiter):
     EXPLOIT_TYPE = ExploitType.BRUTE_FORCE
     _EXPLOITED_SERVICE = "SSH"
 
-    def __init__(self, host):
+    def __init__(self, host, telemetry_messenger: ITelemetryMessenger, options: Dict):
         super(SSHExploiter, self).__init__(host)
         self._update_timestamp = 0
+        self._telemetry_messenger = telemetry_messenger
+        self._options = options
```

Comment:
Is there any reason to initiate these things before the exploiter is run? I don't think there is, so move this logic to the `_exploit_host` function, construct the exploiter object when the plugin is being loaded, and pass the appropriate parameters to this object when the exploiter is actually being run.